app/models/work.rb

```diff
@@ -18,9 +18,7 @@ class Work < ApplicationRecord
   has_many :events, as: :eventable, dependent: :destroy
 
   validates :state, presence: true
-  validates :license, presence: true, inclusion: { in: License.license_list }
-  validates :subtype, work_subtype: true
```

Comment:
I'm not sure why these disappeared in this PR . can they be restored?   Maybe remove in a different PR?

Reply:
Ah, these disappeared because of the conversation we had re: "validation in forms or models" we had on take 1 of the PR. So, I will restore these and push.

Reply:
@jcoyne Fixed. One more look?